### PR TITLE
Tweak text insets to reflect dynamic type.

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.h [hunk 1]
```diff
@@ -66,8 +66,6 @@ typedef NS_ENUM(NSUInteger, OWSMessageGestureLocation) {
 
 @property (nonatomic, nullable, readonly) UIView *bodyMediaView;
 
-@property (nonatomic) BOOL alwaysShowBubbleTail;
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.h]: Removing a vestigial property.

File: Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift [hunk 2]
```diff
@@ -4,6 +4,29 @@
...
+@objc
+public class OWSTextInsets: NSObject {
+
+    @objc public let leading: CGFloat
+    @objc public let trailing: CGFloat
+    @objc public let top: CGFloat
+    @objc public let bottom: CGFloat
+
+    @objc
+    public required init(leading: CGFloat = 0,
```
[the remark this answers is not in the document]
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift]: Done.

**Comment by charlesmchen-signal** [comment]
>I think this needs a rebase, right now I think there's a bunch of redundant changes. 

Done.

PTAL @michaelkirk

File: Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift [hunk 3]
```diff
@@ -40,6 +63,8 @@ public class ConversationLayoutInfo: NSObject {
     @objc public var maxMessageWidth: CGFloat = 0
     @objc public var maxFooterWidth: CGFloat = 0
 
+    @objc public var textInsets = OWSDirectionalEdgeInsets()
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift]: nit: I think a `.zero` class initializer like other edge inset classes would be preferable to default values, no biggie though.

Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift]: Done.

**Pull request closed without merging**